docs/events/timestamps/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Timestamps"
+description: ""
+date: "2025-05-15"
+sidebar_position: 4
+---
+
+Snowplow events have multiple timestamps that are added as the payload moves through the pipeline.
+
+The timestamps are:
+
+| Timestamp             | Added by                                                                            | Description                                                                                                                                                            | In all events |
+| --------------------- | ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `dvce_created_tstamp` | [Tracker](/docs/sources/trackers/index.md)                                          | The device's timestamp when the event was created                                                                                                                      | ✅             |
```

Comment:
I know that below you describe the inaccuracies in this timestamp.  But to end users I think this is a really important point to understand.  Maybe it should be up here in the table too, in case users don't read the detail below the table.

Same for `dvce_sent_tstamp`.